src/main/java/bio/terra/janitor/db/JanitorDao.java

```diff
@@ -288,4 +339,33 @@ public static TrackedResourceAndFlight create(
       return new AutoValue_JanitorDao_TrackedResourceAndFlight(trackedResource, cleanupFlight);
     }
   }
+
+  /** A {@link TrackedResource} and the labels associated with it. */
+  @AutoValue
+  public abstract static class TrackedResourceAndLabels {
```

Comment:
Feels the db objects are used/defined everywhere now.
Shall we make it has a class like TrackedResource, Id, and State?

Reply:
done